nb2kg/nb2kg/managers.py

```diff
@@ -110,14 +117,14 @@ def start_kernel(self, kernel_id=None, path=None, **kwargs):
         if kernel_id is None:
             kernel_name = kwargs.get('kernel_name', 'python3')
             self.log.info("Request new kernel at: %s" % self.kernels_endpoint)
-            response = yield fetch_kg(
-                self.kernels_endpoint,
-                method='POST',
-                body=json_encode({
-                    'name': kernel_name,
-                    'env': {k:v for (k,v) in dict(os.environ).items() if k.startswith('KERNEL_') or k in os.environ['KG_ENV_WHITELIST'].split(",")}
-                })
-            )
+            # try
```

Comment:
is this a start of an incomplete comment or the commenting-out of a `try` statement.  Seems like it should be removed or improved.

Reply:
yup, already removed that on my local changes